BrentOzarULTD/SQL-Server-First-Responder-Kit
# BlitzCache & Blitz - Version checks - 2017 but not yet CU3

Fixes #1528  .

Changes proposed in this pull request:
 - Update BlitzCache so that we don't go looking for columns in DMV's that don't exist
 - Update Blitz to report if this is the case and nudge that they should patch
 - Update Blitz documentation

How to test this code:
 - Find a 2017 server that's not yet on CU3 
 - Run BlitzCache, should now not throw errors
 - Run Blitz, you should see a 100 error showing that user should patch.

Has been tested on (remove any that don't apply):
 - SQL Server 2017 (14.0.1000.169)
